README.md

```diff
@@ -1,16 +1,99 @@
-# RaspBerryDAO Godwoken Bridge - Smart Contracts - Backend(Bridge Service)
+## RaspberryDAO Godwoken Bridge - Smart Contracts - Backend(Bridge Service)
 
-# Sample Hardhat Project
+## Hardhat Project
 
-This project demonstrates a basic Hardhat use case. It comes with a sample contract, a test for that contract, and a script that deploys that contract.
+## RaspberryDAO (Decentralised Automnomous Organisation)
 
```

Comment:
@Mohsinsiddi best to describe the position of this repo in the structure of the entire raspberry dao project, referencing to the other repositories.

Reply:
@Mohsinsiddi this comment hasn't been taken care of yet. Something like this should be in the top areas of the readme for each repo

Structure: (h2)
Repo1 : short description
Repo2 : short description (the repo this readme is with should be bold, for example if this is the structure written in readme of repo2 the repo2 would be bold, )
Repo3 : short description
Repo4 : short description